src/main/java/com/microsoft/spring/data/gremlin/conversion/script/BasicGremlinScriptLiteral.java

```diff
@@ -32,4 +33,18 @@ protected void generateProperties(@NonNull List<String> scriptList, @NonNull fin
             }
         }
     }
+
+    protected String removeIdProperty(@NonNull String scriptLiteral) {
+        final String idProperty = "'id'";
```

Comment:
Is the scriptLiteral of json format? So the id property can only be `'id'` but never `id`?

Reply:
not json, but id is also valid here. will refine it.